partition_version.cc

```diff
@@ -364,6 +369,8 @@ void partition_entry::apply(const schema& s, mutation_partition&& mp, const sche
     }
     new_version->insert_before(*_version);
     set_version(new_version);
+    app_stats.row_insertions = mp.live_row_count(s);
+    app_stats.row_hits = 0;
```

Comment:
I don't understand these two lines. Shouldn't you only be adding to the stats, never clearing them?

Reply:
Indeed, these values should be added, not assigned. Previously `app_stats` was a struct that was returned from this function. It is now corrected.